modules/ROOT/pages/xamarin/security.adoc

```diff
@@ -186,3 +186,23 @@ Dictionary<string, SecurityCheckResult> results = securityService.GetSyncExecuto
 
 The `Builder.WithSecurityCheck` method can be invoked as much as needed to add checks to be executed.
 The value returned by the `Execute` method will be a `Dictionary` whose value is the result of the check, while the key is the ID of the check that produced that result.
+
+[#reporting-selfdefence-metrics]
+=== Reporting Self Defence Checks Results Via the Metrics Service
+In order to report the results of Self Defence utilize the link:./using-metrics-sdk[Metrics] service in conjunction with the `SecurityService`.
```

Comment:
"In order to report the results of Self Defence **checks**, **use**

Metrics **S**ervice

service in conjunction with the `SecurityService`  - should it be 'ISecurityService'